packages/core/src/internal/interfaces/index.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
```

Comment:
Originally I had the motion interfaces in this file. I migrated them into their own directory inside of cds internal